web_src/js/modules/fetch.js

```diff
@@ -14,6 +14,7 @@ function request(url, {headers, data, body, ...other} = {}) {
     } else if (data instanceof URLSearchParams) {
       contentType = 'application/x-www-form-urlencoded';
       body = data;
+      body.set('_csrf', csrfToken);
```

Comment:
It changes the caller's `URLSearchParams` variable.

Reply:
And it doesn't need such `_csrf` parameter. Because the 'x-csrf-token' is enough.

While there is another problem: `GET` request doesn't need `'x-csrf-token'` header.

Reply:
Hmm, there must be some backend code to support the `_csrf` parameter from the forms, I guess this is so HTML forms can use it, but I do agree, in fetch we can set headers so we don't need this.

Reply:
Removed it.